### DISCUSSION TITLE
Error in practical 0301

### DISCUSSION THREAD

**Discussion by Will-214** [IBAHCM/RPiR#231] [Archive]
Hi @richardreeve @soniamitchell due to some sort of bug on my computer when i turned it on this morning it quit the session, sending me back right to the beginning of the practical  0301. So I've been trying to work through it all again.
But when I make a change to the documentation and run "devtools::document()" it gives me this error:
**_Error in step_deterministic_birth_death(latest = tail(population.df, 1),  : 
  could not find function "step_deterministic_birth_death"_**

Would you be able to help me with this?

**Comment by soniamitchell** [comment]
Hey @Will-214, try running `devtools::document()` (to populate your NAMESPACE file if you haven't already), then try installing it with `devtools::install()`, and try again 👍

**Reply by richardreeve** [reply]
I had a look in your repo, but I think you haven't pushed any of the relevant code to it. It might be easier to talk about this in person in the class...
